app/models/user.rb

```diff
@@ -13,13 +13,24 @@ class User < ActiveRecord::Base
          :trackable,
          :validatable
 
-  validates :username, presence: true,
+  validates :username,
+            presence: true,
             uniqueness: true,
             length: { in: 3..20 }
   validates :website, :allow_blank => true,
-            format: { with: URI.regexp },
-            if: Proc.new { |a| a.website.present? }
+            format: {
+              with: URI.regexp },
+              if: Proc.new { |a| a.website.present?
+            }
```

Comment:
`end` at 24, 12 is not aligned with `Proc.new { |a| a.website.present?` at 23, 18 or `if: Proc.new { |a| a.website.present?` at 23, 14